index.html

```diff
@@ -256,32 +256,22 @@ <h3>Full Name</h3>
             <div class="custom-card-wrapper col-12 col-md-6 col-lg-4">
               <div class="custom-card">
                 <div class="avatar-wrapper">
-                  <img src="img/placeholder.jpeg" alt="" class="avatar" />
+         <img src="img/madalin.jpg" alt="" class="avatar" />
```

Comment:
2 issues here:
1. You forgot to add the image 😅 
2. Please add back the spacing (the `img` tag should be visually inside the parent)